python/ray/rllib/agents/qmix/model.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from torch import nn
+import torch.nn.functional as F
+
+
+# TODO(ekl) we should have common models for pytorch like we do for TF
```

Comment:
Is that different from `rllib/models/pytorch/`?

Reply:
Yeah but that's pretty old code. It will probably have to be cleaned up a bunch to get it working properly with the newer APIs.